### PR TITLE
TS-3560: Make slow log overridable.

### PR DESCRIPTION
**Pull request by persiaAziz**
TS-3560 : make slow_log_threshold overridable

### CODE REVIEW

File: plugins/experimental/ts_lua/ts_lua_http_config.c [hunk 1]
```diff
@@ -81,7 +81,8 @@ typedef enum {
   TS_LUA_CONFIG_HTTP_CACHE_FUZZ_PROBABILITY = TS_CONFIG_HTTP_CACHE_FUZZ_PROBABILITY,
   TS_LUA_CONFIG_NET_SOCK_PACKET_MARK_OUT = TS_CONFIG_NET_SOCK_PACKET_MARK_OUT,
   TS_LUA_CONFIG_NET_SOCK_PACKET_TOS_OUT = TS_CONFIG_NET_SOCK_PACKET_TOS_OUT,
-  TS_LUA_CONFIG_LAST_ENTRY = TS_CONFIG_LAST_ENTRY
+  TS_LUA_CONFIG_LAST_ENTRY = TS_CONFIG_LAST_ENTRY,
```
Comment (SolidWallOfCode) [inline plugins/experimental/ts_lua/ts_lua_http_config.c]: This needs to be the last entry.

File: proxy/http/HttpConfig.h [hunk 2]
```diff
@@ -554,8 +554,8 @@ struct OverridableHttpConfigParams {
   MgmtInt flow_low_water_mark;  ///< Flow control low water mark.
 
   MgmtInt default_buffer_size_index;
-  MgmtInt default_buffer_water_mark;
-
+  MgmtInt default_buffer_water_mark;
```
Comment (SolidWallOfCode) [inline proxy/http/HttpConfig.h]: Clean up trailing whitespace.

File: proxy/http/HttpConfig.h [hunk 3]
```diff
@@ -708,7 +708,7 @@ struct HttpConfigParams : public ConfigInfo {
   // Error Reporting //
   /////////////////////
   MgmtByte errors_log_error_pages;
-  MgmtInt slow_log_threshold;
+  //MgmtInt slow_log_threshold;  !@#
```
Comment (SolidWallOfCode) [inline proxy/http/HttpConfig.h]: Remove this entirely, don't leave it commented out.

**Comment by SolidWallOfCode** [comment]
Looks good overall but needs a bit of cleanup. In general the "// !@#" comments need to be removed.

**Comment by persiaAziz** [comment]
Done Syeda Persia Aziz
CSE,BUET 


     On Wednesday, May 27, 2015 3:37 PM, Alan M. Carroll <notifications@github.com> wrote:
   

 Looks good overall but needs a bit of cleanup. In general the "// !@#" comments need to be removed.—

**Pull request closed without merging**